src/styles/app/03-app-shell.css

```
@@ -29,20 +29,97 @@
+}
+
+.glyphToastDescription {
+	color: var(--glyph-toast-muted) !important;
+	font-size: 12.5px !important;
+	line-height: 1.35 !important;
+}
+
+.glyphToastBadge {
+	background: color-mix(
+		in srgb,
+		currentColor 16%,
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)